ml-agents/mlagents/trainers/sac/policy.py

```diff
@@ -206,7 +206,7 @@ def update(
         return update_stats
 
     def update_reward_signals(
-        self, reward_signal_minibatches: Dict[str, Dict], num_sequences: int
+        self, reward_signal_minibatches: Mapping[str, Dict], num_sequences: int
```

Comment:
This was getting Dict[str, AgentBuffer] passed to it, which causes the helpful errors:
```
ml-agents/mlagents/trainers/sac/trainer.py:322: note: "Dict" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance
ml-agents/mlagents/trainers/sac/trainer.py:322: note: Consider using "Mapping" instead, which is covariant in the value type
```